server/sandbox_run.go

```diff
@@ -22,6 +22,11 @@ import (
 	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	// TODO: Remove this const once this value is provided over CRI
+	PodInfraOOMAdj int = -998
```

Comment:
actually, can you just add also this link here https://github.com/kubernetes/kubernetes/issues/47938 so anybody can check in the future